cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -165,25 +165,28 @@
     {# Customized Modernizr build that includes html5shiv.
        Built via gulp-modernizer in `scripts.js` task. #}
     <script src="{{ static('js/modernizr.min.js') }}"></script>
+
+    {#
+      Turn off JavaScript for browsers that don't support ECMAScript 5 features
```

Comment:
Should we change the footer script to do the same instead of relying on the IE 8 comment hack? https://github.com/cfpb/cfgov-refresh/pull/3677/files#diff-89b43ee86bbd4952dc44cf751cfd775cR247

Reply:
Okay check now